plugin/src/withRadarIOS.ts

```diff
@@ -74,6 +74,36 @@ export const withRadarIOS: ConfigPlugin<RadarPluginProps> = (config, args) => {
       },
     ]);
   }
+  if (args.addRadarSDKVerify) {
+    config = withDangerousMod(config, [
+      'ios',
+      async config => {
+        const filePath = path.join(config.modRequest.platformProjectRoot, 'Podfile');
+        const contents = await fs.readFile(filePath, 'utf-8');
+
+        // Check if the pod declaration already exists
+        if (contents.indexOf("pod 'RadarSDK/Verify', '3.19.2-beta.9'") === -1) {
```

Comment:
@lmeier @KennyHuRadar Similar to `addRadarSDKMotion` option. Will we need to update the version by hand in both places?

Reply:
we should probably extract this to a constant in the future. Since we release all the sdks with matching versions (I think, since RadarSDKMotion releases a 3.19 together with a 3.19 main SDK even if there is no change). We can probably just update the version string in the podspec file, and we can read the version from there and use it across the plugin.

Reply:
Yea that's exactly correct — easiest to maintain the same version across the the main module and all submodules.